visualization_log_2022-09-06_08:44:49.txt

```diff
@@ -0,0 +1,7 @@
+2022-09-06 08:44:49,018 - __main__ - INFO - Not all paths are configured in sys.argv, use the paths in xrmocap_data/my_config/shelf_config/shelf_data_visualization_testset.py.
```

Comment:
We do not commit log files. It looks like a mistake in my dataset visualization tool, would you please help me check the tool. It should be written to `logs/visualization_log....txt`.